content/applications/finance/expenses/expense_reports.rst

```diff
@@ -0,0 +1,109 @@
+===============
+Expense reports
+===============
+
+When expenses are ready to submit (such as at the end of a business trip, or once a month), an
+*expense report* needs to be created. Navigate to the main :menuselection:`Expenses` app dashboard,
+which displays a default :guilabel:`My Expenses` view, or go to :menuselection:`Expenses app --> My
+Expenses --> My Expenses`.
+
+Expenses are color coded by status. Any expense with a status of :guilabel:`To Report` (expenses
+that still need to be added to an expense report) the text appears in blue. All other statuses
+(:guilabel:`To Submit`, :guilabel:`Submitted`, and :guilabel:`Approved`) the text appears in black.
+
+Create an expense report
+========================
+
+First, select each individual expense to be added to the report on the :guilabel:`My Expenses`
+dashboard, by ticking the checkbox next to each entry, or quickly select all the expenses in the
+list by ticking the checkbox next to :guilabel:`Expense Date`.
+
+Another way to quickly add all expenses that are not on an expense report is to click
+:guilabel:`Create Report` without selecting any expenses, and Odoo automatically selects all
+expenses with a status of :guilabel:`To Submit` that are not already on a report.
+
+.. image:: expense_reports/create-report.png
+   :align: center
+   :alt: Select the expenses to submit, then create the report.
+
+.. note::
+   Any expense can be selected from the :guilabel:`My Expenses` list, regardless of status. The
+   :guilabel:`Create Report` button is visible as long as there is a minimum of one expense with a
+   status of :guilabel:`To Report` selected. When the :guilabel:`Create Report` button is clicked,
```

Comment:
If you reference the above screenshot, you'll notice that I have not selected any record, and the 'Create Report' button is still visible. So, I am not entirely sure if a minimum of one expense with a 'To Report' status **needs** to be selected in order to see it. Definitely worth a quick double-check to ensure the accuracy of this statement. 👍

Reply:
Thank you- I checked again, and it always appears, unless there's no expense with a status of To Submit or To Report. But just because it is visible, doesn't mean it will work. If all the expenses are on another report (but it hasn't been approved or submitted) you get an error, since there's nothing new to add.